build.properties

```diff
@@ -0,0 +1,45 @@
+ # Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ #
+ # WSO2 LLC. licenses this file to you under the Apache License,
+ # Version 2.0 (the "License"); you may not use this file except
+ # in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #    http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing,
+ # software distributed under the License is distributed on an
+ # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ # KIND, either express or implied. See the License for the
+ # specific language governing permissions and limitations
+ # under the License.
+
+# Product names
+# -----------------------------------------------
+WSO2IS_KM=wso2is-km-5.10.0
```

Comment:
Shall we change these versions to match with OB 3.0.0.

Reply:
Fixed with daf45f713aede96d0fe748349d757bf392889dad